src/modules/Experiences/ExperiencesList/DeleteExperience.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const DeleteExperience = ({ experience, onDelete, classes, handleCancel }) => {
+  const experienceName = `${experience.host} - ${experience.title}`;
+
+  return (
+    <Dialog open={true}>
+      <DialogContent className={classes.dialogContent}>
+        <Typography align="center">
+          Are you sure you want to delete
+          <br />
+          <span className={classes.experienceName}>{experienceName}</span>?
+        </Typography>
+      </DialogContent>
+      <DialogActions className={classes.dialogAction} align="between">
+        <Button
+          className={classes.delete}
+          variant="outlined"
+          onClick={() => onDelete(experience)}
+        >
+          Delete
+        </Button>
+
+        <Button variant="contained" color="secondary" onClick={handleCancel}>
+          No, Go Back
+        </Button>
+      </DialogActions>
+    </Dialog>
+  );
+};
+
+const styles = ({ breakpoints, palette, spacing }) => ({
+  dialogContent: {
+    width: '350px',
+  },
+  dialogAction: {
+    width: '350px',
+    paddingBottom: '20px',
+    display: 'flex',
+    alignSelf: 'center',
+    justifyContent: 'space-between',
+  },
+  delete: {
+    '&:hover': {
+      backgroundColor: '#d10d0d',
```

Comment:
You may want to consider using the theme palette here (e.g. `palette.error.main`) rather than a custom color—this will make it easier to keep the app consistent and also a change a bunch of things all at once.
See: https://material-ui.com/system/palette/

Also, it's worth noting we haven't fully defined the Material design theme yet, so we may end up wanting to tweak error colors in the future and this will make it easier